src/CbcCInterface.jl

```diff
@@ -214,21 +223,33 @@ end
 @getproperty Float64 getObjSense
 
 for s in (:getRowLower, :getRowUpper, :getRowActivity)
-    @eval function ($s)(prob::CbcModel)
+    #This macro creates a unsafe wrapper to the following function
```

Comment:
Misting space after `#` and punctuation at the end of the sentences (http://www.juliaopt.org/JuMP.jl/dev/style/#Comments-1).